WordPress/Classes/Extensions/AbstractPost+fixedMediaLocalURL.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+extension AbstractPost {
+    /// If a post is in the failed state it can contain references to local files.
+    /// When updating the app through the App Store the local paths can change.
+    /// This will fix any outdated local path with the correct one.
+    ///
+    func fixedMediaLocalURL() {
```

Comment:
Grammar nitpick: "fix**ed**" implies to me that it would fix something, and return a corrected version as a return value.

This method however, doesn't return any values and acts on the object itself — I think a better name would be `fixMediaLocalURL()` (or even `fixLocalMediaURLs()`, I think that reads easier)